src/app/shared/components/drawer/content/content.component.ts

```diff
@@ -51,6 +51,7 @@ export class ContentComponent implements OnDestroy {
 
       case 'drawer-initialized':
         this.animationsEnabled = true;
+        this.rightMargin = 456;
```

Comment:
What is the magic number?

Reply:
The drawer content is supposed to resize along with the sidebar but for some reason it always overflowed when I first opened the viewer. So I changed it to manually resize on the first load. Wasn't sure how else to do it.

Reply:
Any thoughts @axdanbol ?

Reply:
I've tested this. It doesn't seem to have any effect. I recommend removal.